frontend/src/Home.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './Home.css';
+
+export default function Home(props) {
```

Comment:
btw, does 'Home' no longer extend Component here? 

previously: `class HomeContent extends Component`

Reply:
Yeah, I didn't think it was necessary for it to be a class Component because it just returns HTML and doesn't have state, so it's a function Component